src/Database/Dialect/SqlserverDialectTrait.php

```diff
@@ -335,32 +335,32 @@ public function schemaDialect(): BaseSchema
     /**
      * Returns a SQL snippet for creating a new transaction savepoint
      *
-     * @param string $name save point name
+     * @param string|int $name save point name
```

Comment:
Why do we need to allow int here?

Reply:
Tests were failing as some of the db implementationa were paasing integers.

Reply:
I see, so casting beforehand is not feasible here.